intro-HPC/ch_fine_tuning_job_specifications.tex

```diff
@@ -351,55 +368,59 @@ \subsection{Number of processors}
 %\shellcmd{qsub -l nodes=N:ppn=M}%
 \end{prompt}
 
-In this example, you request 8 GB of memory in total on the node.
-
 This specifies the number of nodes (nodes=N) and the number of processors per
 node (ppn=M) that the job should use. PBS treats a processor core as a
 processor, so a system with eight cores per compute node can have ppn=8 as its
-maximum ppn request. Note that unless a job has some inherent parallelism of
+maximum ppn request. You can also use ppn=all to request all cores of a node,
```

Comment:
For instance,
qsub -I -l nodes=1;ppn=all
or
qsub -I -l nodes=1;ppn=half
works on UGent systems.

Can not verify with other institutes. Will write as issue.